### PR TITLE
Added My Humpty Wall

### PR DESCRIPTION
**Pull request by saarthakchats**
- [ ] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

### Changes done in this Pull Request
Added my wall
screenshot 2018-10-30 at 2 34 02 pm

- If your change will be reflected on the website, please provide a **Test-Link**
  

- Fixes #<Add issue number here>

### CODE REVIEW

**Comment by saarthakchats** [comment]
@niccokunzmann

**Comment by saarthakchats** [comment]
the rawgit preview link is still not working

File: js/tiles.js [hunk 1]
```diff
@@ -967,7 +967,7 @@ const door = {
         },
         createImages: function() {
           this.wallTop = this.createImage("tiles/rooms/door/yellowDoor.svg");
-          this.wallRight = this.createImage("tiles/rooms/wall/yellowRight.svg");
+          this.wallRight = this.createImage("tiles/rooms/wall/humptywall.svg");
```
Comment (abishekvashok) [inline js/tiles.js]: Maybe add it to the default template?

**Comment by saarthakchats** [comment]
@abishekvashok 
Done

**Comment by niccokunzmann** [comment]
Thanks for your contributions and reviews!

**Pull request merged**